packages/volto/locales/it/LC_MESSAGES/volto.po

```diff
@@ -4911,6 +4921,11 @@ msgstr "Toggle"
 msgid "upgradeVersions"
 msgstr "Aggiorna da versione {origin} a {destination}"
 
+#. Default: "Upload an image from your computer"
+#: components/manage/Widgets/ImageWidget
+msgid "uploadAnImage"
+msgstr ""
```

Comment:
msgstr "Carica un'immagine dal tuo computer"

Reply:
Fine for me